packages/sdk-v2/src/sdk/analytics/amplitude/index.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./Amplitude";
-export { IAmplitudeConfig } from "./types";
+export type { IAmplitudeConfig } from "./types";
```

Comment:
revert. this isn't flow, there's no `export type`, just `export`

Reply:
it throws a compile error on build when there are just export